srcpkgs/m4rie/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'm4rie'
+pkgname=m4rie
+version=20200125
+revision=1
+#archs="i686 x86_64"
+wrksrc=m4rie-20200125
+#create_wrksrc=yes
+build_style=gnu-configure
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="pkg-config autoconf automake libtool"
+makedepends="m4rie"
+depends="m4ri"
+short_desc="M4RIE is a library for fast arithmetic with dense matrices over GF(2^e) for 2 ≤ e≤ 16."
```

Comment:
this does not pass lint, use something like short_desc="Library for fast arithmetic with dense matrices over GF(2^e)"